src/compute/src/server.rs

```diff
@@ -445,26 +445,46 @@ impl<'w, A: Allocate + 'static> Worker<'w, A> {
             return;
         }
 
+        // The last time we did periodic maintenance.
+        let mut last_maintenance = None;
```

Comment:
How important is it that we do a maintenance immediately after startup? Is there even anything to maintain yet?
If we don't need to maintain immediately, initializing this to `Instant::now()` would remove a bit of `Option` handling code below.

Reply:
Yup, that makes it simpler!